### ISSUE TITLE
What's the license of this project?

### ISSUE THREAD

**Issue by ppisar** [jmcnamara/excel-writer-xlsx#285]
Is this piece of software licensed under "GPL-1.0-or-later OR Artistic-1.0-Perl" combination or "Artistic-1.0-Perl" single license?

Many files, e.g. lib/Excel/Writer/XLSX/Chart/Pie.pm, claim:

~~~~
All Rights Reserved. This module is free software. It may be used, redistributed and/or modified under the same terms as Perl itself.
~~~~

According to <https://www.perl.org/about.html>, it is a "GPL-1.0-or-later OR Artistic-1.0-Perl" combination.

But lib/Excel/Writer/XLSX.pm contains a contradicting claim:

~~~~
=head1 LICENSE

The Perl Artistic Licence L<http://dev.perl.org/licenses/artistic.html>.
~~~~

See that the GPL-1.0-or-later part is missing there.

However, a few lines bellow:

~~~~
=head1 COPYRIGHT

Copyright MM-MMXXIII, John McNamara.

All Rights Reserved. This module is free software. It may be used, redistributed and/or modified under the same terms as Perl itself.
~~~~

the same file refers to the combination.

Similarly LICENSE file only quotes Artistic-1.0-Perl license text.

I guess all this confusion comes from your commit 5b04b4c50f824bb6d3f58e2fccb62da3123df7c7 where you deleted the GPL-1.0-or-later part. Is that intentional change of the license, or a mistake?

For me the difference is important, because e.g. for Fedora Linux distribution the sole Artistic-1.0-Perl license is not free enough and without the GPL-1.0-or-later alternative Fedore will have to stop distributing this software.

**Comment by jmcnamara** [comment]
Thanks for pointing this out. The intention is that license is the same as perl so "Artistic License" or "GPL 1.0 or Later". I'll clean this up and clarify. I'll add a SPDX header as well.

**Comment by jmcnamara** [comment]
I've pushed a fix for this to main. You can check to see if it meets your licensing requirements.

**Comment by ppisar** [comment]
Thanks. It looks good now.

**Comment by jmcnamara** [comment]
Do you need me to do a new release so you can package it?

**Comment by ppisar** [comment]
No, I don't need a new release. Your statement about the intended license has been enough.
